modules/channel-web/src/migrations/index.ts

```diff
@@ -0,0 +1,19 @@
+import * as sdk from 'botpress/sdk'
+
+export default class Migration {
+  public async execute(dryRun?: boolean): sdk.MigrationStatus {
```

Comment:
Isn't the proper return type Promise<sdk.MigrationStatus> ?  I'm confused why this builds withtout error

Reply:
That was an example which will be removed, the sdk wasn't built & distributed to modules, so no typings here